interfaces/cython/SConscript

```diff
@@ -15,6 +15,10 @@ build(dataFiles)
 testFiles = localenv.RecursiveInstall("cantera/test/data", "#test/data")
 build(testFiles)
 
+exampleFiles = localenv.RecursiveInstall(
+    "cantera/interfaces/cython/cantera/examples", "#build/python/cantera/examples")
+build(exampleFiles)
```

Comment:
As this is a single file, and it's inside the interface folder already, I think you can just add "cantera/examples/README.txt" into the list of dependencies on line 123, after `setup.py` and `pyproject.toml`. Hopefully that'll do the right thing.